.github/workflows/publish-dist.yml

```diff
@@ -29,9 +29,25 @@ jobs:
       - name: Commit to dist repo
         run: |
           set -ex
+          mkdir dist-repo
           cd dist-repo
+          TARS=(../dist-unpack/cockpit-*.tar.xz)
+          if [ "${#TARS[@]}" -ne 1 ]; then
+              echo "Expected exactly one tarball" >&2
+              exit 1;
+          fi
+          tar --strip-components=1 -xf "${TARS[0]}"
+
+          # prevent any sneaky/buggy stuff from build-dist
+          danger="$(find -maxdepth 1 -name '.git*')"
```

Comment:
If one of the `node_modules` that we happen to ship starts including a `.github` directory one day this is going to trigger.  Hopefully we stop having `node_modules` here by then, though.

Reply:
That's what the `-maxdepth 1` is for. It now ignores somedir/.git* stuff.

Reply:
Still not a huge fan of limiting this instead of just excluding all `.git` stuff.  There were some annoying `.gitattributes` files in some npm modules that it was nice to strip out...

Reply:
We have for example tools/.gitignore which is regularly shipped in the tarball (that's what broke [this run](https://github.com/martinpitt/cockpit/runs/2499583978?check_suite_focus=true.) where I had the "super-strict" mode). I am not actually sure why (and we can clean this up), but indeed some node_modules/somemodule/.github/ seems totally fine to me?